### PR TITLE
Program lecture tracks

### LINKED ISSUE #71: Change program
--- This should be done by either Roy or Daphne --- 
Add current lectures.

### PR DESCRIPTION
**Pull request by royklip**
Fixes #71

### CODE REVIEW

File: src/symposium-homepage.html [hunk 1]
```diff
@@ -130,9 +130,7 @@
 
       <p> On the 1st of May, our symposium about Cyber Security will take place. This symposium, organized by the symposium committee of W.I.S.V. 'Christiaan Huygens' will take place in Theater de Veste, Delft.</p>
 
-      <p>During the day, there will be two separated tracks, one specialized on the mathematical aspects and one specialized on computer science side.</p>
-
-      <p>At the end of the day, we will finish with a discussion about the ethics behind cyber security, followed by drinks and the possibility to network and share thoughts on the discussed topics. </p>
+      <p>During the day, there will be two separated tracks and at the end of the day, we will finish with a discussion about the ethics behind cyber security, followed by drinks and the possibility to network and share thoughts on the discussed topics. </p>
```
Comment (martijnjanssen) [inline src/symposium-homepage.html]: I would change 'two separated tracks' to 'two separate tracks'. There is also a trailing space at the end of the sentence.

File: src/symposium-program.html [hunk 2]
```diff
@@ -64,25 +64,25 @@
...
           </tr>
           <tr>
             <td>10:00</td>
             <td colspan="2">Coffee break</td>
           </tr>
           <tr>
             <td>10:30</td>
-            <td>Lecture track 1</td>
-            <td>Lecture track 2</td>
+            <td>Lecture by Herbert Bos</td>
```
Comment (martijnjanssen) [inline src/symposium-program.html]: Why not mention where these people work?

**Review by martijnjanssen — COMMENTED** [review]
Looks fine, but what is the reason behind removing the TU-Delft logo?

**Comment by TimvdLippe** [comment]
If you want to automatically close a GitHub issue, use `Fixes #XXX` in the pull request description. Simply referencing will not auto-close it.

**Review by TimvdLippe — APPROVED** [review]
Nice responsive work, good job!

**Pull request merged**